frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,40 @@
+import { Heading, Text } from '@bcgov/design-system-react-components';
+import { BaseLayout } from '../components/layout/BaseLayout';
+
+export const Login = () => {
+  return (
+    <BaseLayout>
+      <div className="border-gold mt-[52px] mb-[10px] w-[36px] border-t-[4px]"></div>
```

Comment:
```suggestion
      <div className="border-gold mt-[52px] mb-[10px] w-[36px] border-t-[4px]" />
```
This meets Figma designs exactly 👍 
In the future (not necessary in this PR) I would like to add this as a component that can be called once, just incase it needs to be updated it can be done in one place instead of many.

Reply:
Changes reflected on another branch